protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/mojo/AbstractGenerateMojo.java

```diff
@@ -346,6 +346,18 @@ public AbstractGenerateMojo() {
   @Parameter(defaultValue = DEFAULT_FALSE)
   boolean fatalWarnings;
 
+  /**
+   * Defines the file in which to write the protobin descriptor.
+   *
+   * <p>Leave unspecified to disable. Writes a FileDescriptorSet (a protocol buffer,
+   * defined in descriptor.proto) containing all the input files in
+   * {@code outputDescriptorFile}.</p>
+   *
+   * @since 2.9.0
+   */
+  @Parameter
+  Path outputDescriptorFile;
```

Comment:
There is a small caveat with the Maven API where you _have_ to pass File around, although we convert it internally to work around that... did you happen to run into this while testing?

Reply:
Hi
In my tests, I used the latest version of Maven 3.9.9 and had no issues using Path.

Reply:
Interesting, must just affect nested objects then! All good if it is working, thanks for checking.